### PR TITLE
update caption

### LINKED ISSUE #1: Add collaborators to this repo
## Comment below to be added as a collaborator

We will all be using this repository today. Even though it's public right now, you will not be able to make changes until you're given the correct permissions. We've automated this process via the GitHub API. Once you comment, we will add you as a collaborator.

You'll start to receive a lot of emails. (❗) You can slow these down immediately by clicking the **Unwatch** button at the top of the page and selecting either the **Participating and @mentions** or **Ignore** option.

### PR DESCRIPTION
**Pull request by vvaths**
resolves #1

### CODE REVIEW

File: _slides/10-omarperezrm.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (vyas000) [inline _slides/10-omarperezrm.md:5]: please create a separate branch

File: _slides/02-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Magical New Caption!!!
```
Comment (vyas000) [inline _slides/02-vvaths.md:6]:
```suggestion
Magical AWESOME New Caption!!!!!
```

**Review by vyas000 — CHANGES_REQUESTED** [review]

**Review by vyas000 — APPROVED** [review]
Looks Fine

File: _slides/10-omarperezrm.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-Caption here
+This is fun!
```
Comment (KeyurColpal) [inline _slides/10-omarperezrm.md:7]: Add the New Caption!!!!! @vvaths

**Review by KeyurColpal — APPROVED** [review]

**Review by vyas000 — APPROVED** [review]

**Pull request merged**